### ISSUE TITLE
Can we improve `dist_quantile` printing when 0.5 quantile level is not present?

### ISSUE THREAD

**Issue by dshemetov** [cmu-delphi/epipredict#305]
When 0.5 quantile is not present, the object is printed like this

```r
r$> new_quantiles(c(1,2),c(0.1,0.2))
quantiles(NA)[2]
r$> new_quantiles(c(1,2),c(0.1,0.5))
quantiles(2)[2]
```

Can we improve this somehow?

**Comment by dajmcdon** [comment]
That printing error is a bit more subtle, it's if the median can't reasonably be computed. This currently happens if (a) 0.5 doesn't exist and (b) the available `quantile_levels` don't straddle 0.5.
``` r
new_quantiles(c(1, 2), c(0.1, 0.9))
#> quantiles(1.5)[2]
```

**Comment by dshemetov** [comment]
Gotcha thanks for that detail.
